portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -73,4 +73,12 @@ private int getMaxNumOfComments(HttpServletRequest request) {
     }
     return maxNumComments;
   }
+
+  private Comment entityToCommentConverter(Entity entity) {
```

Comment:
You can move it to Comment class sand make it public and static so that it can be used in other places in future. https://www.baeldung.com/java-constructors-vs-static-factory-methods

Reply:
changed